Header header
string name
low_throughput_hmi/IntMap mapa
geometry_msgs/Pose init_pose
